### ISSUE TITLE
Remove dead/scam project

### ISSUE THREAD

**Issue by batrinuvlad** [forkdelta/tokenbase#1142]
**What token should be updated?**

Address: https://etherscan.io/token/0xe8c5e942b76099c0c6d78271bad3ca002fa7c531
Symbol: HELP
Website: https://www.helpproject.info/

**What about the token should be updated?**
It should be removed for the following reasons:

- No volume on the exchange itself
- Last token transfer since 2 months ago (on etherscan.io)
- No social media activity;
- No public telegram;
- No team section;
- Abstract whitepaper, doesn't describe a platform or a clear implementation path for the "action plan"
- Abstract roadmap;
- No community
- They just stole our ticker and people complain because they could buy a scam coin;

Basically, there isn't any activity for this coin.

**Comment by Kryptoniq** [comment]
FYI, the project did not steal your ticker. The project started before yours.

**Comment by batrinuvlad** [comment]
@Kryptoniq @freeatnet even if it was listed on forkdelta before GoHelpFund, the fact that the project and coin is not active anymore should be a good reason to remove it from the exchange.
